.travis.yml

```diff
@@ -1,5 +1,22 @@
-language: generic
-sudo: required
-services: docker
-script:
-  - bash scripts/run-tests.sh
+matrix:
+  include:
+    - language: python
```

Comment:
Although this is quite nice to speed up TravisCI runs, the documentation tells users to run `scripts/run-tests.sh` and it will not run the mypy test in that case.

You can run mypy in here instead so that it runs both in TravisCI and locally:
https://github.com/release-engineering/estuary-api/blob/9b090856996646294043765b669dd09edb5c0ff4/docker/Dockerfile-tests#L25

If you feel inspired, you could use a matrix and overwrite the run command in the command so that one matrix is for flake8, the other for pytest, and the last one for mypy.